cosmwasm/README.md

```diff
@@ -1,45 +1,93 @@
-# Deploy
+# Pyth CosmWasm
 
-First build the contracts
+This directory contains pyth contract on CosmWasm and utilities to deploy it in CosmWasm chains.
```

Comment:
"the pyth contract". also capitalize Pyth 

this error occurs a few times in the text below as well.